### ISSUE TITLE
no query string for email reset

### ISSUE THREAD

**Issue by single-stop-rj** [graphql-devise/graphql_devise#104]
### Question
hey team. I'm hoping I'm just having a brain fart, but I am working through this email reset. 

```graphql
mutation {
  userSendPasswordReset(email: "test_user-002@exampleuser.com", redirectUrl: "http://localhost:5000/auth") {
    message
  }
}
```

```json
{
errors: [
{
message: "No query string was present"
}
]
}
```
```shell
http://localhost:5000/auth?query=query%28%24token%3AString%21%2C%24redirectUrl%3AString%21%29%7BuserCheckPasswordToken%28resetPasswordToken%3A%24token%2CredirectUrl%3A%24redirectUrl%29%7Bemail%7D%7D&variables%5BredirectUrl%5D=http%3A%2F%2Flocalhost%3A5000%2Fauth&variables%5Btoken%5D=ETsb5W8xDqQQk8CjAiqs
```
decoded
```json
http://localhost:5000/auth?query=query($token:String!,$redirectUrl:String!){userCheckPasswordToken(resetPasswordToken:$token,redirectUrl:$redirectUrl){email}}&variables[redirectUrl]=http://localhost:5000/auth&variables[token]=ETsb5W8xDqQQk8CjAiqs
```
It looks like a query string is in fact present in the uri however. 

Am I just missing something?

**Comment by mcelicalderon** [comment]
Hey @single-stop-rj! I see what's happening. You are passing a redirect url for your `auth` route. The redirect url should actually be a place where you want the user that clicked on the link to land after hitting the API to verify tokens. So you getting this message means it is working. Pass something like http://google.com as a redirectUrl and you'll see how you land on google, but with credentials as query params in the URL. In that route you are supposed to get those credentials to make an authenticated request to the updatePassword mutation, which will allow you to change the password without providing the current password as you clicked on the link.

The process is a bit tricky, we'll try to make it clearer on the readme. DTA has a wiki entry just for this. Also, a new way to do this is supported on the new versions of DTA, I will check if we can implement that.

But this brought another problem to my attention @00dav00, [this](https://github.com/graphql-devise/graphql_devise/blob/55ec2a1e846debf2bed909af2eebab56bd9afe58/app/views/graphql_devise/mailer/reset_password_instructions.html.erb#L5) won't work when you mount the gem in your own schema as the redirect url is fixed to the gem's url

**Comment by single-stop-rj** [comment]
Thank you. It makes sense now! Also happy that you pointed out that it does not yet work in my own schema, as that was the next trip down the hole.

**Comment by mcelicalderon** [comment]
🙌 That will be fixed by #106
